packer/scripts/20-caddy.sh

```diff
@@ -11,8 +12,9 @@ useradd --system \
 
 chown -R caddy:caddy /var/lib/caddy
 
-VERSION_SLUG=$(echo "$CADDY_VERSION" | sed 's/-/_/g' | sed 's/\.[0-9]*\.[0-9]*//g' | sed 's/v//g' | sed 's/\.//g')
-wget -qO /usr/bin/caddy https://github.com/caddyserver/caddy/releases/download/"${CADDY_VERSION}"/caddy"${VERSION_SLUG}"_linux_amd64
+VERSION_SLUG=${CADDY_VERSION//v/}
```

Comment:
I have to admit I do not understand this line

Reply:
It is equivalent to this `echo $CADDY_VERSION | sed 's/v//g'`. I didn't know this syntax, but I use Shellcheck to lint the shell scripts and to warn of possibly risky lines, and it suggested this alternative to sed. I thought it was cool and works as intended.